processor/src/chiplets/aux_trace/mod.rs

```diff
@@ -204,17 +129,20 @@ where
     }
 }
 
-/// Constructs the removals from the table when the hasher absorbs a new sibling node while
-/// computing the new Merkle root.
-fn chiplets_vtable_remove_sibling<E>(main_trace: &MainTrace, alphas: &[E], row: usize) -> E
+// VIRTUAL TABLE RESPONSES
+// ================================================================================================
+
+/// Constructs the inclusions to the table when the hasher absorbs a new sibling node while
+/// computing the old Merkle root.
+fn chiplets_vtable_add_sibling<E>(main_trace: &MainTrace, alphas: &[E], row: usize) -> E
 where
     E: FieldElement<BaseField = Felt>,
 {
-    let f_mu: bool = main_trace.f_mu(row);
-    let f_mua: bool = if row == 0 { false } else { main_trace.f_mua(row - 1) };
+    let f_mv: bool = main_trace.f_mv(row);
```

Comment:
I think `main_trace.f_mv(row)` is inconsistent with the docs? The [docs](https://0xpolygonmiden.github.io/miden-vm/design/chiplets/hasher.html#instruction-flags) say `f_mv == 1` when selectors are `(1, 1, 0)`, but the implementation sets `f_mv == 1` when selectors are `(0, 1, 1, 0)` (one flag is missing from the docs).

Also `f_mva`, `f_mu`, and pretty much all of them are inconsistent with the docs. Unless I'm reading it wrong?

Reply:
The first selector is the chiplet selector flag and is documented in [here](https://0xpolygonmiden.github.io/miden-vm/design/chiplets/main.html#chiplets-order)